### DISCUSSION TITLE
Reboot Grid From SOC

### DISCUSSION THREAD

**Discussion by jaypfe** [Security-Onion-Solutions/securityonion#12317] [Ideas] [answered]
Would be pretty neat to be able to restart the machines that are flagged for one directly from the SOC. 

At least in my case, I have nodes deployed across different server farms, and it becomes just a little bit tedious to restart them at times. 

Not sure if this is possible or not, just thought it would be neat!

**Comment by Mav1814** [comment]
Hi, 

I'm not sure I understand what you mean, but I restart all the machines via salt. In the manager, I do "sudo salt \* cmd.run 'sudo reboot'". And all the machines in the grid are restarted.

cheers
mav

**Comment by TOoSmOotH** [comment, marked as answer]
scroll down in the grid view when you open a node.. You will see a yellow power button highlighted for you to just click and reboot.

**Reply by jaypfe** [reply]
haha, thats awesome! Did not notice that.

Thanks man!
